common/simpcfg.hpp

```diff
@@ -0,0 +1,645 @@
+#pragma once
+
+/**
+ * Provides a simple direct 1-level only config file logic
+ * by Humans for All
+ * 
```

Comment:
I appreciate the simplicity, but we've already surrendered to using `json.hpp` in the examples, so I don't think there is need to add `simpcfg` as well

Reply:
In a common response below, I have indicated the reason why I looked at this simpcfg alternative. Based on your response to same, we can take a call on this aspect.